### DISCUSSION TITLE
How to set sessionoptions for auto-session plugin?

### DISCUSSION THREAD

**Discussion by i11010520** [nix-community/nixvim#2233] [Q&A]
I tried following, not work:
```
programs.nixvim.globalOpts.sessionoptions = [
        "blank"
        "buffers"
        ...
];
```

**Comment by khaneliman** [comment]
Did you try `programs.nixvim.opts` instead of `globalOpts`? The documentation mentions `vim.o` https://github.com/rmagatti/auto-session?tab=readme-ov-file#recommended-sessionoptions-config
